test/run_envoy_bazel_coverage.sh

```diff
@@ -93,6 +93,7 @@ if [[ "$VALIDATE_COVERAGE" == "true" ]]; then
   fi
   COVERAGE_FAILED=$(echo "${COVERAGE_VALUE}<${COVERAGE_THRESHOLD}" | bc)
   if [[ "${COVERAGE_FAILED}" -eq 1 ]]; then
+      echo "##vso[task.setvariable variable=COVERAGE_FAILED]$COVERAGE_FAILED"
```

Comment:
```suggestion
      echo "##vso[task.setvariable variable=COVERAGE_FAILED]${COVERAGE_FAILED}"
```

per bash style guide, ditto below.

Reply:
done, thank you